README.md

```diff
@@ -39,6 +39,12 @@ The site should now be live at `http://localhost:4000/capital-framework/`.
 $ grunt vendor
 ```
 
+**Deny/Allow connections.** If you are an employee of the CFPB, you may encounter the following message:
```

Comment:
:+1: Sounds good, I've pushed an update. I'd not seen that dialog box before using the CFPB laptop, is it a Yosemite thing?